src/cljam/algo/convert.clj

```diff
@@ -13,7 +13,8 @@
             [cljam.io.util :as io-util]
             [com.climate.claypoole :as cp])
   (:import [java.nio ByteBuffer]
-           [cljam.io.fastq FASTQRead]))
+           [cljam.io.fastq FASTQRead])
+  (:refer-clojure :exclude [chunk seq]))
```

Comment:
`chunk` in this ns can be renamed to `chunk'` or something.

https://github.com/chrovis/cljam/blob/eb2e396c84ef45813a5d3806df07dbe0a27c73b5/src/cljam/algo/convert.clj#L33-L37

The same for `seq`.
https://github.com/chrovis/cljam/blob/eb2e396c84ef45813a5d3806df07dbe0a27c73b5/src/cljam/algo/convert.clj#L120

```clj
  [{:keys [flag qual qname] ^String seq' :seq}]
```

In general, I prefer renaming symbols over `:refer-clojure :exclude` because the latter requires programmers to memorize the global state in each ns while reading/modifying codes.